### ISSUE TITLE
Error in running docs website

### ISSUE THREAD

**Issue by cseas** [JedWatson/react-select#4232]
I just made a fresh clone of the react-select repository and tried to run the docs but I keep getting a webpack error.

Steps to reproduce:
```sh
git clone https://github.com/JedWatson/react-select
cd react-select/docs
npm install
npm start
```

Error message produced:
```sh
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./index.html] 1.75 KiB {0} [built]
    [./node_modules/lodash/lodash.js] 530 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 472 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
ℹ ｢wdm｣: Failed to compile.
```

Operating system: Ubuntu 20.04.1 LTS

**Comment by Rall3n** [comment]
@cseas The docs come without a compiled `react-select` build. You have to run the `build` task in the projects root directory before running the `start` task. Alternatively you can run the `watch` task in parallel.

**Comment by cseas** [comment]
@Rall3n I went to the root folder and ran the following commands:
```
npm install
npm run build
```
After the build finished, I went back to the `docs/` folder and ran `npm start`.

The error changed slightly but still the same thing:
![Selection_089](https://user-images.githubusercontent.com/29686866/94995463-15a41f00-05bc-11eb-8724-237951a224cc.png)

**Comment by Rall3n** [comment]
My mistake. I mistook the docs with the test suite.

But I could reproduce your problem. Seems like the environment does not get correctly set up using `npm`. But it works using `yarn`.

I would suggest trying it again with `yarn`.

**Comment by cseas** [comment]
@Rall3n I installed `yarn` and tried running the docs website with it. It seemed to work fine, although I did get a message about linux incompatibility. That's possibly a separate issue so I'll close this one.
